content/tutorials/manage-the-rippled-server/configuration/run-rippled-in-reporting-mode.md

```diff
@@ -0,0 +1,375 @@
+
+* PostgreSQL database to hold relational data, which is used mainly by [tx method][] and [account_tx method][]. 
+
+When a reporting mode server receives an API request, it loads the data from these data stores if possible. For requests that require data from the P2P network, the reporting mode forwards the request to a P2P server, and then passes the response back to the client.
+
+Multiple reporting mode servers can share access to the same network accessible databases (PostgreSQL and Cassandra); at any given time, only one reporting mode server writes to the databases, while all the others read from the databases. 
+
+## How to Run Reporting Mode
+
+### Prerequisites
+
+1. Ensure that your system meets the [system requirements](system-requirements.html).
+
+    **Note:** If you choose to use Cassandra as the database, the disk requirements for `rippled` will be lower as the data will not be stored on your local disk.  
+
+2. In order to run reporting mode, you also need to run one or more `rippled` servers in [P2P mode](install-rippled.html). Ensure that you have at least one `rippled` server running in P2P mode. 
+
+3. Install and configure the datastores required to run `rippled` in reporting mode. 
+    
+    1. Install Postgres. 
+
+        <!-- MULTICODE_BLOCK_START -->
+
+        *Linux*
+
+        1. Download and install PostgreSQL on Linux. For instructions, refer to https://www.postgresqltutorial.com/install-postgresql-linux/.
+        
+        2. Connect to the PostgreSQL Database Server using psql, and create a user `newuser` and a database `reporting`.
+
+        psql postgres
+	        CREATE ROLE newuser WITH LOGIN PASSWORD ‘password’;
+            ALTER ROLE newuser CREATEDB;
+        \q
+        psql postgres -U newuser
+        postgres=# create database reporting;
+
+        *Windows*
+
+        1. Download and install PostgreSQL on Windows. For instructions, refer to https://www.postgresqltutorial.com/install-postgresql/.
+        
+        
+        2. Launch the psql program and connect to the PostgreSQL Database Server, and create a user `newuser` and a database `reporting`. 
+            psql postgres
+	        CREATE ROLE newuser WITH LOGIN PASSWORD ‘password’;
+            ALTER ROLE newuser CREATEDB;
+        \q
+        psql postgres -U newuser
+        postgres=# create database reporting;
+
+        *macOS*
+
+        1. Download and install Postgres on macOS.  
+
+        brew install postgres
+        brew services start postgres
+
+        2. Connect to the PostgreSQL Database Server using psql and create a user `newuser` and a database `reporting`.
+
+        psql postgres
+	        CREATE ROLE newuser WITH LOGIN PASSWORD ‘password’;
+            ALTER ROLE newuser CREATEDB;
+        \q
+        psql postgres -U newuser
+        postgres=# create database reporting;
+
+     <!-- MULTICODE_BLOCK_START -->
```

Comment:
The end of a multicode block should be marked with `<!-- MULTICODE_BLOCK_END -->` (not `START` again)